user-access-service/src/main/java/no/unit/nva/database/IdentityService.java

```diff
@@ -34,8 +34,10 @@ static IdentityService defaultIdentityService(DynamoDbClient dynamoDbClient) {
     UserDto addUser(UserDto user) throws ConflictException;
     
     void addRole(RoleDto roleDto) throws ConflictException, InvalidInputException;
-    
-    void updateUser(UserDto user) throws NotFoundException;
+
+    void updateRole(RoleDto roleDto) throws InvalidInputException, NotFoundException;
```

Comment:
This is added just to make it possible from test. Currently, we have no API to manage roles. They are handled as system data.

Reply:
I was about to ask why you deleted updateUser, but lo and behold, below it was.